### ISSUE TITLE
Documentation of INC algorithm

### ISSUE THREAD

**Issue by CinnamonRolls1** [CinnamonRolls1/daa-project#19]
Use a README.md file placed within the INC folder. Functions and variables must be defined in a way similar to the greedy documentation. </br>
**Comment below if you're taking up this issue so as to avoid duplicate attempts.**

**Comment by CinnamonRolls1** [comment]
Actually, in hindsight, it's better if everyone just adds the description of the function they defined. I'll add a README.md file.

**Comment by safeer1999** [comment]
Yeah its better if we people just add the comments for their own function
and please mention the return type of the returned object

**Comment by CinnamonRolls1** [comment]
@kishanrakesh plis add documentation for update_m

**Comment by CinnamonRolls1** [comment]
@kainspraveen add documentation for update__L_i

**Comment by CinnamonRolls1** [comment]
This is still incomplete.
